test/unit/org/apache/cassandra/cql3/functions/OperationFctsTest.java

```diff
@@ -33,6 +33,20 @@
 
 public class OperationFctsTest extends CQLTester
 {
+
+    @Test
+    public void testStringConcatenation() throws Throwable
+    {
+        createTable("CREATE TABLE %s (a text, b ascii, c text, PRIMARY KEY(a, b, c))");
```

Comment:
I would add one UDT to be on the safe side as sometimes those behave in funny ways.

Reply:
a pointer in that direction...

Reply:
I guess that what @bereng is suggesting is to add a user type with `createType(firstname text, surname text)` and to do a concat using `column['firstname'] + column['surname']`.

Reply:
@bereng After thinking about it there are no interactions possible between UDT and the new function. By consequence I do not believe that adding a test with some make sense here.